proto/aggkit/prover/v1/aggchain_proof_generation.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package aggkit.prover.v1;
+
+// Service for fetching aggchain proof.
+service AggchainProofService {
+  // Fetches a aggchain proof for a given start_block.
+  rpc FetchAggchainProof(FetchAggchainProofRequest) returns (FetchAggchainProofResponse);
```

Comment:
I'm not sure about the method name, `Fetch` feels like the proof is already there.
Maybe we could iterate later over that with something like `GenerateAggchainProof` or `RequestProof`, as the service is already named `AggchainProofService`.

Reply:
renamed to `GenerateAggchainProof`